pkg/webservice/handlers_test.go

```diff
@@ -1720,3 +1720,83 @@ func verifyStateDumpJSON(t *testing.T, aggregated *AggregatedStateInfo) {
 	assert.Check(t, len(aggregated.Config.SchedulerConfig.Partitions) > 0)
 	assert.Check(t, len(aggregated.Config.Extra) > 0)
 }
+
+func TestCheckHealthStatusNotFound(t *testing.T) {
+	NewWebApp(&scheduler.ClusterContext{}, nil)
+	req, err := http.NewRequest("GET", "/ws/v1/scheduler/healthcheck", strings.NewReader(""))
+	assert.NilError(t, err, "Error while creating the healthcheck request")
+	resp := &MockResponseWriter{}
+	checkHealthStatus(resp, req)
+
+	var errInfo dao.YAPIError
+	err = json.Unmarshal(resp.outputBytes, &errInfo)
+	assert.NilError(t, err, unmarshalError)
+	assert.Equal(t, http.StatusNotFound, errInfo.StatusCode, statusCodeError)
+	assert.Equal(t, "The healthy status of scheduler is not found", errInfo.Message, jsonMessageError)
```

Comment:
Could you change this text to sth else? It sounds funny.
I propose "Health check is not available".

Reply:
Already addressed comments in the latest commit, many thanks !